cli/crates/server/src/bridge/server.rs

```diff
@@ -14,14 +14,15 @@ use common::environment::Environment;
 
 use sqlx::query::{Query, QueryAs};
 use sqlx::{migrate::MigrateDatabase, query, query_as, sqlite::SqlitePoolOptions, Sqlite, SqlitePool};
+use std::collections::HashMap;
 use std::net::{Ipv4Addr, SocketAddr};
 use std::sync::Arc;
 
 use tokio::sync::broadcast::Sender;
 use tower_http::trace::TraceLayer;
 
 async fn query_endpoint(
-    State(pool): State<Arc<SqlitePool>>,
+    State((_port, _environment_variables, pool)): State<(u16, HashMap<String, String>, Arc<SqlitePool>)>,
```

Comment:
Not important, but a `State<SomeStruct>` might have been more concise here?

Reply:
Absolutely. But, in fact, these changes weren't supposed to be part of this PR, I failed at splitting my branch. So for now I took this out altogether. In the future PR we'll have a struct for this.